rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RealPreference.java

```diff
@@ -80,7 +79,9 @@
   }
 
   @Override public void delete() {
```

Comment:
Does that make sense at all? Isn't the whole synchronicity passé through the usage is apply?